src/kanata/mod.rs

```diff
@@ -345,6 +349,11 @@ impl Kanata {
             dynamic_macros: Default::default(),
             log_layer_changes,
             caps_word: None,
+            movemouse_inherit_accel_state: cfg
```

Comment:
This needs to be repeated in `do_live_reload` as well. Unfortunately I haven't changed it to have a nice way to deduplicate the code between the initial create and live reloading.